origin-js/src/services/discovery-service.js

```diff
@@ -126,10 +128,16 @@ class DiscoveryService {
     if (opts.idsOnly) {
       return resp.data.listings.nodes.map(listing => listing.data.id)
     } else {
-      return resp.data.listings.nodes.map(listing => listing.data)
+      return resp.data.listings.nodes.map(listing => this.aggregateListingData(listing))
     }
   }
 
+  aggregateListingData(listingNode) {
```

Comment:
nit: this is more "flattening" rather than aggregating. maybe "flattenListingData" would be more representative of what this method does ? and since it is private and does not need to be exposed, prefix with "_". So perhaps call it _flattenListingData.

supernit: move it at top of the file since in general it improves readability to have the code for a function first before it gets used.

Reply:
👍 agreed